### PR TITLE
OSS CI Export

### PR DESCRIPTION
**Pull request by winkingturtle-vmw**
- Add bin/tests.bash
- Ensure DOCKER_REGISTRY_* env variables are set
- ioutil->os package rename

### CODE REVIEW

File: metrics/metrics.test
Comment (MarcPaquette) [inline metrics/metrics.test:1]: Was this file accidentally committed?

File: store/filesystems/loopback/loopback.test
Comment (MarcPaquette) [inline store/filesystems/loopback/loopback.test:1]: Test file needed?

File: store/filesystems/mount/mount.test
Comment (MarcPaquette) [inline store/filesystems/mount/mount.test:1]: Test file needed?

**Review by MarcPaquette — CHANGES_REQUESTED** [review]
Looks like a few test binaries were included in the commit. Can we get those fixed?
Thanks

File: metrics/metrics.test
Comment (winkingturtle-vmw) [reply metrics/metrics.test:1]: Yes, it was committed by mistake a while back. `.test` files should not be needed

**Review by MarcPaquette — APPROVED** [review]

**Pull request merged**